recipes/go/build.sh

```diff
@@ -0,0 +1,24 @@
+# First, build go1.4 using gcc, then use that go to build go>1.4
+# Should I make this a separate package called go-bootstrap?
+# Should I check the checksum of this tarball? What about windows?
+mkdir go-bootstrap && pushd $_
+curl -L https://storage.googleapis.com/golang/go1.4-bootstrap-20170531.tar.gz | tar -xzf -
+export GOROOT_BOOTSTRAP=$PWD/go
+cd $GOROOT_BOOTSTRAP/src
+./make.bash
+
+# Should I run all.bash instead? That will run tests too, but some tests
+# like unshare() require higher capabilities which might not be available
+# on the CI systems
+pushd $SRC_DIR/src
+./make.bash
+
+rm -fr $GOROOT_BOOTSTRAP
+mkdir $PREFIX/go
+cp -rv $SRC_DIR/* $PREFIX/go/
+
+pushd $PREFIX/bin
+for binary in ../go/bin/*; do ln -s $binary ; done
+
+# I am not setting GOROOT to $PREFIX/go in an activate script here, as it
+# doesn't really seem to be required. But should I do that anyway?
```

Comment:
FWIR one does need to set `GOROOT`. This has been the recommended advise (unless something has changed recently).

Also I do recall having certain subtle bugs if `GOROOT` was not set. Would have to remember what they were though... 😅

Reply:
Agreed. I've added that in the updated recipe.